### PR TITLE
Map Stripe to Solaris topup status

### PR DESCRIPTION
**Pull request by p-janik**

### CODE REVIEW

File: src/routes/topUps.ts [hunk 1]
```diff
@@ -8,12 +8,25 @@ import { getStripeClient } from "../helpers/stripe";
...
 
 const log = getLogger("topUps");
 
 const TOP_UP_CHECK_DELAY_IN_MS = 3000;
 
+const mapTopUpStatus = (status: Stripe.PaymentIntent.Status): TopUpStatus => {
+  switch (status) {
+    case "canceled":
+      return TopUpStatus.CANCELLED;
+    case "processing":
```
Comment (lera) [inline src/routes/topUps.ts:21]: last time I've fetched not confirmed topup, I got status `requires_payment_method`. Maybe this one means ACCEPTED

Comment (lera) [reply src/routes/topUps.ts:21]: https://stripe.com/docs/payments/paymentintents/lifecycle#intent-statuses

Comment (p-janik) [reply src/routes/topUps.ts:21]: hmm but we create topup only after clicking the button, not on the card screen

Comment (p-janik) [reply src/routes/topUps.ts:21]: should we then map all of these `require_` to ACCEPTED?

Comment (lera) [reply src/routes/topUps.ts:21]: I think so

**Review by lera — APPROVED** [review]

**Pull request merged**